docs/sgp40/raspberrypi/README.md

```diff
@@ -1 +1,68 @@
 # SGP40 on Raspberry Pi
+
+This document explains how to set the SGP40 up to run with a Raspberry Pi. This became very easy with the new SGP40 since it can run with a logic level of 3.3v which matches the one the Raspberry Pi has. 
+
+## Getting started
+
+### Connecting the Sensor
+
+Your Sensor should have four different connectors: VCC, GND, SDA, SCL. Please refere to other documentation of the Sensor to find out which connector is which on your specific modelle.
+Use the following pins to connect your SGP40:
+
+| *SGP40*        | *Raspberry Pi*  | 
+|----------------|-----------------|
+| VCC            | Pin 1           |
```

Comment:
could you add a description of the pin to make it easier to understand when using a different board, e.g.

```suggestion
| VCC            | Pin 1  (3.3V source)  |
```